.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
```

Comment:
I'm a bit puzzled why this isn't kicking off an action on the PR. Actions are enabled for the project 🤔

Reply:
Might need explicit trigger event. I added such in latest commit

Reply:
Might also be that workflow yml file has to be exist on the main branch first. Here you see successful execution for my fork: https://github.com/MarcoEidinger/ReleaseNotes/actions/runs/1743616272

Reply:
Yeah, I was wondering if it needs to land in main first, although I don't recall that being the case.

Reply:
Right, it's because the action is linked to your fork. I've just pushed your branch into this repo and it kicked off.

In the SPI Server project we've got

```
on: [push]
```

I'm not sure if that would make a difference, because it might mean that actions won't run on PRs unless they're pushed into this repo?

Reply:
No, such events get executed i forks. Scheduled workflows are disabled by GitHub for forks (but can be enabled).